### DISCUSSION TITLE
"Good first issues" are not displayed in the "contribute" area of the repository

### DISCUSSION THREAD

**Discussion by svlad-90** [community/community#22699] [New to GitHub]
<p>Does anyone have the idea, why exactly I do not have “good first issues” listed in the “contribute” area of my repository?</p>
<hr>
<p>Non-closed good first issues available: <a href="https://github.com/svlad-90/DLT-Message-Analyzer/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22" rel="nofollow noopener">https://github.com/svlad-90/DLT-Message-Analyzer/issues?q=is%3Aissue+is%3Aopen+label%3A"good+first+issue"</a></p>
<hr>
<p>Contribute area is empty despite the fact that issues were created ~2 weeks ago: <a href="https://github.com/svlad-90/DLT-Message-Analyzer/contribute" rel="nofollow noopener">https://github.com/svlad-90/DLT-Message-Analyzer/contribute</a></p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/1/1fa060bffbd71d499fe78eae138c5276cb7347a4.png" data-download-href="/uploads/short-url/4vMpoW5yKS9F2DRvP5ZcSIFPm9C.png?dl=1" title="image" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/4373469/181072635-11ca4d91-2049-4abb-8e3a-4c0c4186af1e.png" alt="image" data-base62-sha1="4vMpoW5yKS9F2DRvP5ZcSIFPm9C" width="690" height="114" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/1/1fa060bffbd71d499fe78eae138c5276cb7347a4_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">image</span><span class="informations">692×115 6.94 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<hr>
<p>May be I should configure something to have it filled in?</p>

**Comment by yamiacat** [comment]
<p>Hi there! :wave: Welcome to the Community!</p>
<p>Hmmm, they certainly should be showing up at the contribute URL from what I can see. I’ve queued a job to reindex the issues of the repository, which should have completed after 24 hours.</p>
<p>If they’re still not showing up, we’ll have to investigate further!</p>
<p>(Kudos to getting your project to 100% community health, by the way - that’s great!)</p>

**Comment by svlad-90** [comment]
<p>Hi <a class="mention" href="/yamiacat">@yamiacat</a>, thanks for kind words regarding my attempts to build a contributable repository.<br>
Also thanks for queueing re-indexing of issues. Unfortunately, it still does not work. The “Contribute” area is empty.</p>
<p>There is a specific point about my repository, which might be related to this issue. Some time ago I’ve renamed it from the “DLTMessageAnalyzer” to “DLT-Message-Analyzer”. Could it be, that renaming has such a side-effect on the Github’s parsing sequence? Or, might it be, that the new name ( which contains “-” characters ) contradicts to some internal parsing rules?</p>
<p>Besides that, I do not have other ideas regarding why it is happening.</p>

**Comment by yamiacat** [comment]
<p>Hi <a class="mention" href="/svlad-90">@svlad-90</a> - I’ve been digging into this, and discovered that unfortunately the system that populates the “contribute” area is currently offline due to that section of our backend having a major overhaul in progress! It’s been down since May.</p>
<p>Issues that were picked up before that are still displayed in the “contribute” sections of their repos, but at the moment new issues are not being shown there. I don’t have a timeline for when this feature will start working again yet, but I’ll update you once I know!</p>

**Comment by svlad-90** [comment]
<p>Hi <a class="mention" href="/yamiacat">@yamiacat</a>, thank you very much for the update!</p>
<p>For me, it was  most important, that:</p>
<ul>
<li>I was doing everything correctly</li>
<li>GitHub’s support is aware of the identified issue</li>
<li>others won’t waste their time figuring out why it currently does not work</li>
</ul>
<p>Looking forward to get further updates from your side, once it is working again :slight_smile:</p>

**Comment by yamiacat** [comment, marked as answer]
<p><a class="mention" href="/svlad-90">@svlad-90</a> - This is now working again! :tada:</p>

**Comment by svlad-90** [comment]
<p>Hi <a class="mention" href="/yamiacat">@yamiacat</a>, I confirm - feature works now.<br>
Thanks a lot for your support! ))</p>
